GOT-StudentVersion/GOT-StudentVersion/EpisodeDetailViewController.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  EpisodeDetailViewController.swift
+//  GOT-StudentVersion
+//
+//  Created by Jack Wong on 8/19/19.
+//  Copyright © 2019 C4Q . All rights reserved.
+//
+
+import UIKit
+
+class EpisodeDetailViewController: UIViewController {
+
+    @IBOutlet weak var photo: UIImageView!
+    @IBOutlet weak var seasonTitle: UILabel!
+    @IBOutlet weak var episodeNumber: UILabel!
+    @IBOutlet weak var runtimeLabel: UILabel!
+    @IBOutlet weak var airDateLabel: UILabel!
+    @IBOutlet weak var episodeTitle: UILabel!
+    
+    @IBOutlet weak var summaryTextView: UITextView!
+    
+    var delegate: GOTEpisode?
```

Comment:
delegate is not a good name for this variable. Delegate usually is used when we have a protocol, call this episode or something